src/lib/bluetooth/bluetoothbackend.cpp

```diff
@@ -36,6 +36,14 @@ BluetoothBackend::BluetoothBackend(QObject *parent)
 {
```

Comment:
Noted for the future, but remember all those sub-commits will get squashed when merged into master.